x-pack/legacy/plugins/apm/public/hooks/useAvgDurationByCountry.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+// import { useRef } from 'react';
+import { useFetcher } from './useFetcher';
+import { useUrlParams } from './useUrlParams';
+import { loadAvgDurationByCountry } from '../services/rest/apm/transaction_groups';
+
+export function useAvgDurationByCountry() {
+  const {
+    urlParams: {
+      serviceName,
+      start,
+      end /* transactionName, transactionType*/
```

Comment:
do these comments mean that these parameters will get implemented later, or will they be removed?